src/tic_tac_toe/protocol/player.clj

```diff
@@ -1,7 +1,7 @@
 (ns tic-tac-toe.protocol.player)
 
 (defprotocol PlayerProtocol
-	(next-move [player board ]))
+	(next-move [player game ]))
 
```

Comment:
You have a bunch of extra spaces at the end of this file, also after `game` in your argument list.